File: RepoTransBench/source_projects/Python/sbidy_pywizlight/pywizlight/public_tests/test_public_scenes.py
from pywizlight.scenes import get_scene_name_from_id

def test_scene_ids_public():
    # Use an uncommon/edge ID and confirm it returns None or similar
    assert get_scene_name_from_id(256) is None
    # Check another valid but different scene id for name
    assert get_scene_name_from_id(18) == "Candlelight"

File: RepoTransBench/target_projects/Python/Rust/sbidy_pywizlight/public_tests/test_public_scenes.rs
use pywizlight_rust::scenes::get_scene_name_from_id;

#[test]
fn test_scene_ids_public() {
    // Uncommon/edge id
    assert!(get_scene_name_from_id(256).is_none());
    // known
    assert_eq!(get_scene_name_from_id(18), Some("Candlelight"));
}